docs/src/tutorials/getting_started/debugging.jl

```diff
@@ -0,0 +1,270 @@
+#     section of the Gurobi documentation, along with the
+#     [Debugging numerical problems](https://yalmip.github.io/inside/debuggingnumerics/)
+#     section of the YALMIP documentation.
+
+# ### Common sources
+
+# Common sources of numerical issues are:
+#
+#  * Very large numbers and very small numbers as problem coefficients. Exactly
+#    what "large" is depends on the solver and the problem, but in general,
+#    values above 1e6 or smaller than 1e-6 cause problems.
+#  * Nonlinear problems with functions that are not defined in parts of their
+#    domain. For example, minimizing `log(x)` where `x >= 0` is undefined when
+#    `x = 0` (a common starting value).
+
+# ### Strategies
+
+# Strategies to debug sources of numerical issues include:
+#
+#  * Rescale variables in the problem and their associated coefficients to
+#    make the magnnitues of all coefficients in the 1e-4 to 1e4 range. For
+#    example, that might mean rescaling a variable from measuring distance in
+#    centimeters to kilometers.
+#  * Try a different solver. Some solvers might be more robust than others for a
+#    particular problem.
+#  * Read the documentation of your solver, and try settings that encourage
+#    numerical robustness.
+#  * Set bounds or add constraints so that all nonlinear functions are defined
+#    across all of the feasible region. This particularly applies for functions
+#    like `1 / x` and `log(x)` which are not defined for `x = 0`.
+
+# ## Debugging an infeasible model
+
+# A model is infeasible if there is no primal solution that satisfies all of
+# the constraints. In general, an infeasible model means one of two things:
+#
+#  * Your problem really has no feasible solution
+#  * There is a mistake in your model.
+
+# ### Example
+
+# A simple example of an infeasible model is:
+
+model = Model(HiGHS.Optimizer)
+set_silent(model)
+@variable(model, x >= 0)
+@objective(model, Max, 2x + 1)
+@constraint(model, 2x - 1 <= -2)
+
+# because the bound says that `x >= 0`, but we can rewrite the constraint to be
+# `x <= -1/2`. When the problem is infeasible, JuMP may return one of a number
+# of statuses. The most common is `INFEASIBLE`:
+
+optimize!(model)
+termination_status(model)
+
+# Depending on the solver, you may also receive `INFEASIBLE_OR_UNBOUNDED` or
+# `LOCALLY_INFEASIBLE`.
+
+# A termination status of `INFEASIBLE_OR_UNBOUNDED` means that the solver could
+# not prove if the solver was infeasible or unbounded, only that the model does
+# not have a finite feasible optimal solution.
+
+# Nonlinear optimizers such as Ipopt may return the status `LOCALLY_INFEASIBLE`.
+# This does not mean that the solver _proved_ no feasible solution exists, only
+# that it could not find one. If you know a primal feasible point, try providing
+# it as a starting point using [`set_start_value`](@ref) and re-optimize.
+
+# ### Common sources
+
+# Common sources of infeasibility are:
+#
+#  * Invalid mathematical formulations
```

Comment:
This is better off as the last bullet point, e.g., "Otherwise invalid mathematical formulations"